__tests__/app.test.js

```diff
@@ -454,3 +451,18 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("Should return a JSON object containing info for all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const { api } = response.body;
+        expect(api).not.toBe(undefined);
```

Comment:
As you're testing api["GET /api"].description, this expect(api).not.toBe(undefined) is a bit unnecessary

Reply:
Oh yeah, very true! Thanks